packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -22,67 +21,31 @@ const buy_sell_filters = [
 
 class BuySell extends Component {
     state = {
-        buy_sell_list: [],
-        is_loading   : true,
-        selected_ad  : {},
-        show_popup   : false,
-        filter_value : 'buy',
+        filter_value: 'buy',
+        selected_ad : {},
+        show_popup  : false,
     }
 
     componentDidMount() {
         // Call API to retrieve buy_sell ad list
-        this.setState({
-            buy_sell_list: [
-                ...this.state.buy_sell_list,
-                {
-                    id             : 'buy_id',
-                    advertiser     : 'John doe',
-                    country        : 'Indonesia',
-                    currency       : 'IDR',
-                    type           : 'buy',
-                    asset          : 'USD',
-                    fix_price      : 12000,
-                    amount         : 20,
-                    min_transaction: 15000,
-                    max_transaction: 240000,
-                    payment_method : 'Bank transfer',
-                    advertiser_note: 'Whatsapp: +60182655318 please send to Maybank 239847238947 JOHN DOE',
-                },
-                {
-                    id             : 'sell_id',
-                    advertiser     : 'John doe',
-                    country        : 'Indonesia',
-                    currency       : 'IDR',
-                    type           : 'sell',
-                    asset          : 'USD',
-                    fix_price      : 12000,
-                    amount         : 20,
-                    min_transaction: 15000,
-                    max_transaction: 240000,
-                    payment_method : 'Bank transfer',
-                    advertiser_note: 'please contact through Whatsapp: +60182655318',
-                },
-            ],
-            is_loading: false,
-        });
+        this.setState({ is_loading: false });
     }
 
-    handleClick = (ad) => {
-        this.setState({ selected_ad: ad, show_popup: true });
+    setSelectedAd = selected_ad => {
```

Comment:
```suggestion
    setSelectedAd = (selected_ad) => {
```

Reply:
@mahdiaryayi-fs currently our codebase has both styles for function parameters and until we have a eslint rule for this I think this is fine